src/content/docs/alerts-applied-intelligence/new-relic-alerts/rest-api-alerts/alerts-conditions-api-field-names.mdx

```diff
@@ -146,9 +146,11 @@ Not every field listed in this glossary is required for every condition type. Th
     id="fill_option"
     title="fill_option"
   >
-    For sporadic data, you can avoid false alerts by filling the gaps (empty windows) with synthetic data. The default is `None`.
+    For sporadic data, you can avoid false alerts by filling the gaps (empty windows) with synthetic data. 
 
-    The other values are `Last known value` and `Custom static value`.
+    * `none`: (Default) Use this if you don’t want to take any action on empty aggregation windows. On evaluation, an empty aggregation window will reset the threshold duration timer. For example, if a condition says that all aggregation windows must have data points above the threshold for 5 minutes, and 1 of the 5 aggregation windows is empty, then the condition won’t be in violation.
+    * `static`: Use this if you’d like to insert a custom static value into the empty aggregation windows before they’re evaluated. This option has an additional, required parameter of fillValue (as named in the API) that specifies what static value should be used. This defaults to 0.
```

Comment:
What defaults to 0, static or fillValue?

Reply:
@urbiz-nr, `fillValue`, which defaults to 0, supplies its value to `static`. So, when we say `static` defaults to zero, it is just inheriting the default 0 value from `fillValue`, unless you change `fillValue`. Does that help?

Also, I added styling around `fillValue`.